krabs/krabs/filtering/event_filter.hpp

```diff
@@ -123,9 +145,9 @@ namespace krabs {
             return;
         }
 
-        if (!predicate_(record)) {
-            return;
-        }
+		if (predicate_ != nullptr && !predicate_(record)) {
```

Comment:
Tabs are being used here instead of spaces.

Reply:
Seems already done by previous fixes